Bottom: a4bec4881425e1d7de04b377801291e965ac59fb
Top:    a287e451b32bda1abf8d58b4111e105c8271b5f4
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-12 11:01:32 +0800

Refresh of vhost_net-introduce-1

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 02eb629..6df0c4a 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -489,7 +489,7 @@ static bool vhost_has_more_pkts(struct vhost_net *net,
 				struct vhost_virtqueue *vq)
 {
 	return !vhost_vq_avail_empty(&net->dev, vq) &&
-	       likely(!vhost_exceeds_maxpand(net));
+	       likely(!vhost_exceeds_maxpend(net));
 }
 
 /* Expects to be always run from workqueue - which acts as
@@ -585,7 +585,7 @@ static void handle_tx(struct vhost_net *net)
 		}
 		total_len += len;
 		if (total_len < VHOST_NET_WEIGHT &&
-		    vhost_has_more_pkts(net, vq))
+		    vhost_has_more_pkts(net, vq)) {
 			msg.msg_flags |= MSG_MORE;
 		} else {
 			msg.msg_flags &= ~MSG_MORE;
